Headers/Simperium_OSX.h

```diff
@@ -6,7 +6,7 @@
 //  Copyright © 2016 Simperium. All rights reserved.
 //
 
-#import <UIKit/UIKit.h>
+#import <Foundation/Foundation.h>
```

Comment:
Evidently, despite this being called `_OSX`, it was just a copy of the `_iOS` version.

But now that we actually use it as the umbrella header for the macOS build, we cannot import UIKit or the view controller files.